P/Powsybl/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+using BinaryBuilder, Pkg
+
+name = "Powsybl"
+version = v"0.1.0"
+sources = [
+    GitSource("https://github.com/powsybl/powsybl.jl.git", "0a9e3110c366808e7ac23cfd9e19f51793807392"),
+    GitSource("https://github.com/powsybl/pypowsybl.git", "cd5fea41bbfb2897fd71a6e63b2d07a465055699")
+]
+
+julia_versions = [v"1.7", v"1.8", v"1.9", v"1.10"]
```

Comment:
I believe the recommendation is to use `julia_version` coming from the `include("../../L/libjulia/common.jl")` below, any reason not to do that?

Reply:
I began writing my script from outside the source tree and stuck with that. So no specific reason, I can change it.